app/assets/images/payment_icons/ing-homepay.svg

```diff
@@ -0,0 +1,9 @@
+  <path fill="#FF5300" d="M34 6h4v8h-4z"/>
+  <use fill="transparent" stroke="rgba(0, 0, 0, 0.07)" stroke-miterlimit="10" stroke-width="2"/>
```

Comment:
I don't see the transparent border. Could you and try to follow similar pattern like Shopify Pay?

https://github.com/activemerchant/payment_icons/blob/efd33be748023e67e7b25c7e483277247a13f604/app/assets/images/payment_icons/shopify_pay.svg